src/frontend/src/styles/main.css

```diff
@@ -387,10 +385,21 @@ body {
 a,
 .t-link {
   display: inline;
-  color: var(--rc-interaction-text);
   color: inherit;
-  text-decoration: underline;
   cursor: pointer;
+  background-image: linear-gradient(
+    to right,
+    var(--vc-brand-pink),
+    var(--vc-brand-pink) 50%,
+    #000 50%
+  );
+  background-size: 200% 100%;
+  background-position: -100%;
+  position: relative;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  transition: all 0.2s ease-in-out;
```

Comment:
I would specify what changes. All is quite CPU/GPU intensive and some times comes with side-effects.

Reply:
sure